packages/helper-backend/src/litellm/litellm.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Body, Controller, Param, Post, Get, Put, Delete, NotFoundException } from '@nestjs/common';
+import { LiteLLMService } from './litellm.service';
+import { CreateNewModel } from './dtos/litellm.dto';
+
+@Controller('litellm')
```

Comment:
I'm not sure we want to expose making a model through here. At this point we are just proxying the LiteLLM backend. I think it makes more sense to just call the LiteLLM service when we make the LangFlow mapping